client/src/components/Search/Search.js

```diff
@@ -33,7 +28,9 @@ export default class Search extends React.Component {
                 });
             }
             if (didChange) {
-                document.querySelector('.search-column').scrollTo({top: 0});
+                if(document.getElementById('search-column')) {
+                    document.querySelector('search-column').scrollTo({top: 0});
```

Comment:
This doesn't make any sense, it was selecting a class not an id, and your second selector is missing the `.` before `search-column`

```
let searchColumn = document.querySelector('.search-column');
if (searchColumn) searchColumn.scrollTo({top: 0});
```